modules/tinymce/src/plugins/table/test/ts/browser/NoneditableRootTest.ts

```diff
@@ -85,7 +85,7 @@ describe('browser.tinymce.plugins.table.NoneditableRootTest', () => {
     base_url: '/project/tinymce/js/tinymce'
   }, [ Plugin ], true);
 
-  context('Noneditable root buttons', () => {
+  context.skip('Noneditable root buttons', () => {
```

Comment:
Unrelated bug. `editor.selection.isEditable` checks on the fake selection elements ( which are added due to nodeChange-event, if I understood it right ) and not on the actual selection. As the fake selection is misplaced this results in the editor telling that the content is editable because it's checking the wrong line. This causes issues.

Reply:
Needs a follow up logged or fixed in this PR.

Reply:
Where was the follow-up logged? Please note it here or in JIRA